services/bepro.ts

```diff
@@ -27,7 +27,7 @@ export default class BeproService {
     });
     // this.bepro.start();
     this.network = new Network({
-      contractAddress: `0x67d98dc9a353b72493c987d4d9abaf5114271675`,
+      contractAddress: "0xDf956E90a6E1c2BE14FB0c694389a0071EB6Eb91",
```

Comment:
This service will disappear, as in: we will not use it, and will delete it by next monday but I wouldn't mind having the same contract address you chose to `bepro-service` be the same